src/compiler/parser.ts

```diff
@@ -2164,17 +2168,24 @@ namespace ts {
             return allowIdentifierNames ? parseIdentifierName() : parseIdentifier();
         }
 
-        function parseTemplateExpression(): TemplateExpression {
+        function parseNoSubstitutionTemplate(isTaggedTemplate?: boolean) {
+            if (isTaggedTemplate) {
+                reScanTemplateHeadOrNoSubstitutionTemplate();
+            }
+            return <NoSubstitutionTemplateLiteral>parseLiteralNode();
+        }
+
+        function parseTemplateExpression(isTaggedTemplate?: boolean): TemplateExpression {

```

Comment:
This is always called with `true`, so I don't think it needs the optional argument.